src/main/kotlin/ro/cadeca/weasylearn/persistence/user/UserRepository.kt

```diff
@@ -0,0 +1,5 @@
+package ro.cadeca.weasylearn.persistence.user
+
+import org.springframework.data.repository.CrudRepository
+
+interface UserRepository : CrudRepository<UserEntity, Long>
```

Comment:
Add new line at end of file

Intellij settings -> Editor -> general --> (scroll down) Save Files: Check option "Ensure an empty line at end of a file on Save"